docs/examples/setup-symfony.rst

```diff
@@ -46,9 +46,8 @@ It will be ready in seven simple steps:
 2. Create a new VirtualHost directory
 3. Install Symfony
 4. Symlink webroot directory
-5. Enable Symfony prod (``app.php``)
-6. Setup DNS record
-7. Visit http://my-symfony.loc in your browser
+5. Setup DNS record
```

Comment:
Can you also adjust the line on line 43:

```
- It will be ready in seven simple steps:
+ It will be ready in six simple steps:
```

Reply:
Ok